deployment/production/ingress.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
The production ingress is not the same as the local/test/acc.
It requires a different approach with the tls which we can do later on.

Reply:
OK